pages/features.html

```diff
@@ -112,289 +142,65 @@
-      </p>
-
-      <p><b>{% trans "Advanced Controller Scripting Engine" %}</b><br>
-        {% blocktrans with midi_scripting_wiki_link="/wiki/doku.php/midi_scripting" %}Mixxx features a cutting-edge JavaScript-based scripting engine for maximum flexibility. MIDI or HID messages can be mapped onto custom functions which execute complex behaviour. <a href="{{ midi_scripting_wiki_link }}">Read more</a>{% endblocktrans %}
+        {% trans "Song gain is automatically adjusted when loading a deck so your mixes maintain a consistent volume. Mixxx reads existing ReplayGain tags and analyzes songs that don't have them." %}
       </p>
     </div>
-
     <div class="halfbox_right">
-      <div style="height: 200px;">
-        <center><img src="{% static '/static/images/feature_vinylcontrol.png' %}"></center>
-      </div>
-
-      <H2>{% trans "Free Timecode Vinyl Control" %}</H2>
+      <h2><img src="{% static '/static/images/ic_preferences_soundhardware.png' %}" class="feature_icon">{% trans "Quad Microphone Input" %}</h2>
       <p>
-        <b>{% trans "Vinyl jockeys love it" %}</b><br>
-
-        {% trans "Control Mixxx from your existing turntables or CDJs and a mixer using timecoded vinyl or CDs.  We're the only free timecode vinyl control software for Windows, Mac OS X, and Linux, so we made sure to do it right." %}
-      </p>
-
-      <p>
-        {% blocktrans with vinyl_control_wiki_link="/wiki/doku.php/vinyl_control" %}Serato CV02, Traktor Scratch MK1, and MixVibes DVS V2 timecodes are supported with absolute positioning, needle cueing, and end-of-record detection. Mixxx even works with the soundcard you already have! <a href="{{ vinyl_control_wiki_link }}">Read more</a>{% endblocktrans %}
+        {% trans "Whether you're dropping thick reggae vocals over some dub, giving shoutouts on the air, or throwing a rap battle, integrate live input from microphones into your mix." %}
       </p>
     </div>
 
     <div style="clear: both;"></div>
-
+    <h1>{% trans "Native Language Support" %}</h1>
+    <p>
+      {% blocktrans with com_transifex_mixxxdj_link="https://www.transifex.com/organization/mixxx-dj-software" %} Use Mixxx in Spanish, Catalan, French, German, Italian, Russian, Finnish, Czech, Dutch, Polish, or Japanese... and more! See our <a href="{{ com_transifex_mixxxdj_link }}">Translation overview</a> page.{% endblocktrans %}
+    </p>
+    
     <div class="gapfiller" style="height: 57px">
       {% include "download_button.html" %}
-      <p style="text-align: center;">{% trans "Don't wait - Download Mixxx and start DJing now for FREE:" %}</p>
+      <p style="text-align: center;">{% trans "Download Mixxx and start DJing now for FREE:" %}</p>
     </div>
-
-    <div id="full_specs">
-      <a name="full_specs"></a>
-      <H2>{% trans "Full Feature List" %}</H2>
```

Comment:
We should have at least *some* bullet points.  I agree the existing list is excessive but it helps to show that mixxx isn't just another half-ass open source project